imagedephi/main.py

```diff
@@ -24,6 +27,11 @@
 
 shutdown_event = asyncio.Event()
 
+fomat_message = (
```

Comment:
fomat -> format

Reply:
fixed  in b6e8bb9f84efa681723dffea0853dce933bf1fda